lib/dor/workflow/client/lifecycle_routes.rb

```diff
@@ -10,33 +10,31 @@ def initialize(requestor:)
         end
 
         # Returns the Date for a requested milestone from workflow lifecycle
-        # @param [String] repo repository name
         # @param [String] druid object id
         # @param [String] milestone_name the name of the milestone being queried for
         # @param [Number] version the version to query for
         # @param [Boolean] active_only (false) if true, return only lifecycle steps for versions that have all processes complete
         # @return [Time] when the milestone was achieved.  Returns nil if the milestone does not exist
         #
-        def lifecycle(repo, druid, milestone_name, version: nil, active_only: false)
```

Comment:
This isn't a backward compatible change.  You need to be able to accept the old parameters and warn when the old parameters were passed.

Reply:
Ah, yeah, I'll update today.